.github/workflows/testing-and-deployment.yml

```
@@ -26,10 +26,10 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - { os: macos-12, python-version: "3.9" } # intel processor
-          - { os: macos-13, python-version: "3.10" } # intel processor
-          - { os: macos-14, python-version: "3.11" } # arm processor
-          - { os: macos-latest, python-version: "3.12" } # arm processor
+          - { os: macos-13, python-version: "3.9" } # intel processor
```

Should we also remove this  comment?
```suggestion
          - { os: macos-13, python-version: "3.9" }
```